### ISSUE TITLE
Support for bash script before running

### ISSUE THREAD

**Issue by caioreis** [iron-meteor/iron-cli#94]
Hi,

In some projects we need to create custom start scripts to verify some things, or just to pull some codes inside private packages for example. It would be nice to have this option with iron, not just sourcing some sh.

Tks

**Comment by cmather** [comment]
agreed this is a good idea. perhaps sketch out some code in the lib/commands/run.js command and ask people to take a look.

**Comment by caioreis** [comment]
I'm on it! (:
